src/workflows/server/abstract_workflow_store.py

```diff
@@ -23,6 +23,7 @@ class PersistentHandler(BaseModel):
     workflow_name: str
     status: Status
     ctx: dict[str, Any]
+    handler_metadata_json: str | None = Field(default=None)
```

Comment:
why the nested json? Can't this just be a `dict[str, Any]` (and default to `{}`)?